x-pack/test/security_solution_endpoint/apps/endpoint/endpoint_list.ts

```diff
@@ -90,8 +89,6 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
     describe('when there is data,', () => {
       before(async () => {
-        const endpointPackage = await policyTestResources.getEndpointPackage();
```

Comment:
Why were these removed?
We need the package in order to set the transform frequency to run faster so that these tests don't' take a long time.

Reply:
@kevinlog recommended removing these to diagnose a previous issue in CI, but they can be replaced at this point. I'll revert.

Reply:
sounds good. Thanks @kpollich

Reply:
@kpollich @paul-tavares thanks for resolving this.  I originally suggested that we remove this line because I thought the test was failing on a timeout waiting for the Endpoint package to install.

This line still doesn't do everything we think it does since I think it only reduces the frequency of one transform.  There is still the `united` transform to reduce, so this line was speeding things up as fast as it originally did.  If this wasn't the root cause of the test failure, it's OK to add back in.

Thanks again for looking into it!

Reply:
True @kevinlog . I think the more important thing to us, which I realized just now, is that its checking that the correct versioned transform is running (by taking the package version).